src/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Solution</title>
+</head>
+<body>
+<h1>Universal Declaration of Human Rights</h1>
```

Comment:
don't forget about whitespaces
```suggestion
  <h1>Universal Declaration of Human Rights</h1>
```